products/facilities/facdb/sql/doe_universalprek.sql

```diff
@@ -2,41 +2,41 @@ DROP TABLE IF EXISTS _doe_universalprek;
 SELECT
     uid,
     source,
-    name as facname,
-    parsed_hnum as addressnum,
-    parsed_sname as streetname,
-    address,
-    NULL as city,
-    zip as zipcode,
+    sitename AS facname,
+    parsed_hnum AS addressnum,
+    parsed_sname AS streetname,
+    siteaddress AS address,
+    NULL AS city,
+    state AS zipcode,
     boro,
-    NULL as borocode,
-    NULL as bin,
-    NULL as bbl,
+    NULL AS borocode,
+    NULL AS bin,
+    NULL AS bbl,
     (CASE
-        WHEN type = 'DOE' THEN 'DOE Universal Pre-K'
```

Comment:
The query relies on the `type` column to produce facdb classifications. The values in the `type` column have changed since the dataset's last version, but the mapping of old to new values appears to be 1-1. 

I kept old values in the query (for example: `WHEN type = <old_val> OR type = <new_val> ...`) in case we would like to build FacDB using an older version of this dataset.